README.md

```diff
@@ -506,6 +506,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [Metalama](https://www.postsharp.net/metalama) - Only for C#. Metalama generates the boilerplate of the code on the fly during compilation so that your source code remains clean. It is free for open-source projects, and its commercial-friendly free tier includes three aspects.
   * [tabnine.com](https://www.tabnine.com/) — Tabnine helps developers create better software faster by providing insights learned from all the code in the world. Plugin available.
   * [v0.dev](https://v0.dev/) — v0 uses AI models to generate code based on simple text prompts. It generates copy-and-paste friendly React code based on shadcn/ui and Tailwind CSS that people can use in their projects. Each generation takes at minimum 30 credits. You start up with 1200 credits, and get 200 free credits every month.
+  * [Bitloops](https:bitloops.com/) — Bitloops is a dedicated Frontend Co-Pilot that helps developers with all the tasks when converting design into code. It analyzes the entire design and generates well-structured, component-based, professional code in minutes. It integrates directly into Figma and your IDE. Free tool to extract images, text, color palette and CSS. Credits required for more complex code generation. 
```

Comment:
pls fix the url, more importantly we need to know from the site that it will have a free tier post launch which isnt clear atm